templates/elasticsearch.yml.j2

```diff
@@ -11,6 +10,10 @@ cluster.name: elasticsearch
 node.name: {{inventory_hostname}}
 {% endif %}
 
+{% if (es_version is version_compare('7.0.0', '>=')) %}
+cluster.initial_master_nodes: {{inventory_hostname}}
```

Comment:
I don't think this one should be here. We expect users to configure discovery themselves https://github.com/elastic/ansible-elasticsearch/#basic-elasticsearch-configuration